### PR TITLE
Add getting URL from git config

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: gitio.zsh [hunk 1]
```diff
@@ -14,21 +14,35 @@ function gitio() {
...
+    printf "USAGE: gitio [[%s]%s/%s] [%s] \n" \
       "https://github.com/" \
       "${fg[cyan]}username${reset_color}" \
       "${fg[magenta]}repo${reset_color}" \
       "${fg[green]}vanity code${reset_color}" \
   }
 
+  # Get URL from git config
+  if [[ "$#" -eq 0 && -s ".git/config" ]]; then
+    local CONFIG_URL="$(
```
Comment (denysdovhan) [inline gitio.zsh:26]: This is amazing idea! But there is a problem with ssh:

image

Comment (ghost) [inline gitio.zsh:26]: I'll fix it later

**Comment by ghost** [comment]
@denysdovhan so what do you think about this?

**Comment by denysdovhan** [comment]
Nice! 
Thank you!

**Pull request merged**